### PR TITLE
[T4A6][W11-B2]Tan Wang Leng

### PR DESCRIPTION
**Pull request by yamgent**

### CODE REVIEW

**Comment by yamgent** [comment]
Ready for review

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:3]: no \* imports. use eclipse shortcut to fix these.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:11]: add comments.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    
+    @Test
+    public void isAnyNull_nullInListInput_returnsTrue() {
+        assertTrue(Utils.isAnyNull("Hello", null, "!"));
+    }
+
+    @Test
+    public void elementsAreUnique_nullInput_returnsTrue() {
+        assertTrue(Utils.elementsAreUnique(null));
```
Comment (cmkumar87) [inline test/java/seedu/addressbook/common/UtilsTest.java:34]: areElementsUnique_nullInput_returnsTrue  is a better name.

**Comment by cmkumar87** [comment]
@yamgent review comments and close PR.

**Pull request closed without merging**